Sources/AblyAssetTrackingPublisher/PublisherBuilder.swift

```diff
@@ -67,4 +67,9 @@ public protocol PublisherBuilder {
      Sets an optional `logHandler` that can capture logs from all across the sdk
      */
     func logHandler(handler: LogHandler?) -> PublisherBuilder
+    
+    /**
+     Sets an optional `vehicleProfile`for the publisher, Currently there are two vehicle profiles : Bicycle and Car
```

Comment:
Super minor comment - between `vehicleProfile` and `for` there's a space missing.

Reply:
387c37efca13803f3336c12543b267e8fc217151